mobile/lib/services/hive_service.dart

```diff
@@ -182,11 +203,12 @@ class HiveService {
 
 class HiveBox {
   static String get appNotifications => 'appNotifications';
+  static String get searchHistory => 'searchHistory';
   static String get kya => 'kya';
   static String get profile => 'profile';
   static String get encryptionKey => 'hiveEncryptionKey';
   static String get analytics => 'analytics';
-  static String get airQualityReadings => 'airQualityReadingsBox';
-  static String get nearByAirQualityReadings => 'nearByAirQualityReadings';
+  static String get airQualityReadings => 'airQualityReadings-v1';
+  static String get nearByAirQualityReadings => 'nearByAirQualityReading-v1';
```

Comment:
Creating new hive storage boxes for air quality readings. Previous boxes got corrupted when we removed hardcoded regions in [this PR.](https://github.com/airqo-platform/AirQo-frontend/pull/979#discussion_r1033732029) hence causing the app to crush on startup. @Baalmart